### ISSUE TITLE
DPK Modifications (Square Enix Archive) Crashes on Real Hardware (3DS)

### ISSUE THREAD

**Issue by Oersted-LAL** [FanTranslatorsInternational/Kuriimu2#217]
Game Name: Final Fantasy 20th Anniversary Edition
Console: 3DS
Format extension(s): DPK, PCK
Type: Archive

Bug details: Repacking the DPK causes the game to crash on real hardware when a modified file is loaded. (Ex. When JOB_NORMAL is modified, only when JOB_NORMAL is loaded, i.e. in combat with an unpromoted unit, the game will crash. The DPK File also becomes slightly larger after repacking, from 109,293KB to 110,073KB.)

Sample files (if possible): https://mega.nz/file/ifgWUbSJ#MeMD1sRvpH5ByLzhYBga3y-Wa_zz6QmkHy9ZGHrxPIw
A Modified version of JOB_NORMAL, with a sprite change over JOBB00.BCTEX. Replacing the original version with this crashed the game.

**Comment by Oersted-LAL** [comment]
Crown.DPK in a ZIP file, as requested by onepiecefreak on Discord.
https://mega.nz/file/zSQglDiB#RkWHDocEO2UwBelIyQqbDmpdMW5Gl0xNi0t9518Hh1I

**Comment by Oersted-LAL** [comment]
Latest Build fixed issue.

If you want to successfully alter a file in the DPK, Extract that file, and open it as a DPK in Kuriimu2. Replace the file, and save it as a seperate file. Reopen Crown.DPK, and replace the file, save as a seperate file. Should work for any altered file. Here is proof it works.
![2021-12-20_12-53-45 269_top](https://user-images.githubusercontent.com/86030820/146824825-f4c14bda-5498-4800-958a-32ed1e2c1a8f.png)


EDIT: May be wrong, as Kuriimu2 can alter files while they are open, and save them properly. This is just the way I did it, and the way I got it to work.
